### ISSUE TITLE
Invalid DropDown component value causes infinite WebSocket connections

### ISSUE THREAD

**Issue by gitars** [ToolJet/ToolJet#9196]
### Version Information

v2.28.0

### Environment

Docker, self-hosted, ECS.

### What is the expected behaviour?

The editor should not crash.

### What is the current behaviour?

Infinite websocket connections causing the browser tab to crash and the editor being unusable.

### How to reproduce the issue?

1. Add a dropdown component
2. Set the default value to `{{[2]}}`. The UI should complain, but keep the value and click away
3. The editor becomes unusable. Or refresh the page, and the editor is unusable. Infinite websocket requests.
4. Since the editor is unusable, you cannot directly fix the issue in the UI. 

### Screenshots or Screencast

![Screenshot 2024-03-25 at 12 53 01 PM](https://github.com/ToolJet/ToolJet/assets/4552296/17ba32ad-e76c-46b4-b58e-f48123a4bf7a)

### Please provide any traces or logs that could help here.
From the Chrome console, many logs saying:

```
main.js?a154613a3b248a34d52b:115 WebSocket connection to 'wss://myhost.com/ws' failed: Insufficient resources
```

### Any possible solutions?

Not sure.

### Can you identify the location in the driver source code where the problem exists?

🤷 

### If the bug is confirmed, would you be willing to submit a PR?

No, sorry.

**Comment by gsmithun4** [comment]
Fix released on [v2.33.2](https://github.com/ToolJet/ToolJet/releases/tag/v2.33.2)

**Comment by nakulnagargade** [comment]
@GrowwithNaveen Closing this issue as this is resolved in the above mentioned PR.
